### PR TITLE
fixed absolute dating on comments as well

### PR DESCRIPTION
**Pull request by robertleeshook**

### CODE REVIEW

File: app/views/tags/comment.html [hunk 1]
```diff
@@ -7,7 +7,26 @@
 		<div class="comment-author">              
       <a href="@{Application.news(_comment.author.id)}">${_comment.author.name}</a>
     </div>
-		<div class="comment-date">${_comment.updatedAt.format('dd MMM yy')}</div>
+		<div class="comment-date">
+		#{if (new Date().getTime() - _comment.updatedAt.getTime())/1000/60 < 1}
+        	less than a minute
+        #{/if}
+        #{elseif (new Date().getTime() - _comment.updatedAt.getTime())/1000/60 < 60}
```
Comment (jtessler) [inline app/views/tags/comment.html:14]: If we're going to use this, we should probably export this to a separate file instead of copying the code.

**Comment by amshali** [comment]
related to #145

**Comment by robertleeshook** [comment]
thats fine

File: app/views/tags/comment.html (hunk 1 above)
Comment (amshali) [inline app/views/tags/comment.html:14]: I second that.

**Comment by amshali** [comment]
I will merge this, but will open a new issue for refactoring.

**Comment by amshali** [comment]
I think we might wanna use this jQuery plugin:
http://timeago.yarp.com/

**Pull request merged**